cse-cc-import-runner-app/src/test/java/com/farao_community/farao/cse/import_runner/app/dichotomy/DichotomyResultHelperTest.java

```diff
@@ -79,7 +80,7 @@ void testLimitingElement() {
         Mockito.when(pstChanger2.getAllSteps()).thenReturn(mockedSteps);
         String limitingElement = dichotomyResultHelper.getLimitingElement(dichotomyResult, network);
 
-        assertEquals("cnec1prevId", limitingElement);
+        Assertions.assertThat(limitingElement).isIn("cnec1prevId", "cnec1outageId");
```

Comment:
We haven’t received confirmation to change this test, and it’s not related to this topic